plugins/tracers/gstsharktracer.c

```diff
@@ -249,6 +256,10 @@ gst_shark_tracer_finalize (GObject * object)
   g_hash_table_unref (priv->myhooks);
 
   G_OBJECT_CLASS (gst_shark_tracer_parent_class)->finalize (object);
+  g_shark_tracer_refcount--;
+  if (g_shark_tracer_refcount == 0) {
```

Comment:
[g_atomic_int_dec_and_test](https://developer-old.gnome.org/glib/stable/glib-Atomic-Operations.html#g-atomic-int-dec-and-test)

Reply:
done!